core/src/test/scala/org/apache/spark/ExternalShuffleServiceSuite.scala

```diff
@@ -255,4 +255,22 @@ class ExternalShuffleServiceSuite extends ShuffleSuite with BeforeAndAfterAll wi
       }
     }
   }
+
+  test("SPARK-38640: using external shuffle service fetching disk persisted blocks") {
```

Comment:
I'm wondering if this `disk persisted blocks` correct for this test case.

Reply:
I can make it more descriptive

Reply:
Updated the test name